examples/booking_app/lib/feature_hotel/search/blocs/hotel_list_bloc.dart

```diff
@@ -30,20 +39,23 @@ abstract class HotelListEvents {
 
 abstract class HotelListStates {
   @RxBlocIgnoreState()
-  Stream<PaginatedList<Hotel>> get searchedHotels;
+  Stream<PaginatedList<Hotel>> get hotels;
 
   @RxBlocIgnoreState()
-  Stream<DateTimeRange?> get dateRangeFilter;
+  Stream<String> get hotelsFound;
 
   @RxBlocIgnoreState()
   Stream<String> get queryFilter;
 
-  /// Returns when the data refreshing has completed
   @RxBlocIgnoreState()
-  Future<void> get refreshDone;
+  Stream<DateRangeFilterData> get dateRangeFilterData;
```

Comment:
@DDavidPrime do we really need to use @RxBlocIgnoreState() here ?

Reply:
Not really. Updated!